docs/graphcast/radios/poi-radio.md

```diff
@@ -145,9 +161,53 @@ RUST_LOG="warn,hyper=warn,graphcast_sdk=debug,poi_radio=debug"
 
 ## Monitoring the Radio
 
-If the Radio operator has set up a Slack Bot integration and the Radio finds a POI mismatch, it sends alerts to the designated channel. The operator can also inspect the logs to see if the Radio is functioning properly, if it's sending and receiving messages, if it's comparing normalised POIs, if there is a found POI mismatch, etc.
-
-Support for Prometheus metrics and Grafana dashboards is coming soon!
+### Notifications
+
+If the Radio operator has set up a Slack Bot and/or Discord bot integration and the Radio finds a POI mismatch, it sends alerts to the designated channels. The operator can also inspect the logs to see if the Radio is functioning properly, if it's sending and receiving messages, if it's comparing normalised POIs, if there is a found POI mismatch, etc.
+
+### Prometheus & Grafana
+
+The POI Radio exposes metrics that can then be scraped by a Prometheus server and displayed in Grafana. In order to use them you have to have a local Prometheus server running and scraping metrics on the provided port. You can specify the metrics host and port by using the environment variables `METRICS_PORT` and `METRICS_HOST`. We also provide a [Grafana dashboard config JSON file](https://github.com/graphops/poi-radio/blob/main/poi-radio-grafana.json) which you can use to visualise the metrics in Grafana.
+
+## HTTP Server
+
+If you set the `SERVER_HOST` and `SERVER_PORT` environment variables, the Radio will spin up an HTTP server with a GraphQL API. Currently the Radio supports the following routes:
+
+- `/health` for health status
+- `/metrics` for http service metrics about the server
```

Comment:
Maybe we could add a sentence to clarify that `/metrics` is different from `METRICS_PORT:METRICS_HOST` and provide different data, that no radio specific information  is exposed, aside from HTTP requests at `SERVER_HOST:SERVER_PORT`?

Reply:
Hmmm makes sense, will make an issue for this fix

Reply:
so do we need more docs clarification for it now given that a new issue will be created to track it (and it will require new docs changes) ?

Reply:
I would say we can remove the server's `/metrics` descriptions from the docs as if they do not exist; should be a quick fix on the radio

Reply:
I would say we can remove the server's `/metrics` descriptions from the docs as if they do not exist; should be a quick fix on the radio

Reply:
removed